<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindow</class>
 <widget class="QWidget" name="SearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchWindow</string>
  </property>
  <widget class="QListWidget" name="answers_list_widget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_line_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="matching_question_line_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>search_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SearchWindow</receiver>
   <slot>update_results()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_results()</slot>
 </slots>
</ui>
